workers/Worker_cross_origin_security_err.htm

```diff
@@ -11,7 +11,18 @@
       assert_true(e instanceof Event);
     });
   } catch (e) {
```

Comment:
You can remove the try/catch entirely; `async_test` will handle any exceptions better than you can.

Reply:
Thanks! But it looks like we're actually going to need to keep the try/catch after all, since the test can still pass in the catch